source/concepts/erasure-coding.rst

```diff
@@ -48,6 +48,14 @@ Use the MinIO `Erasure Code Calculator
 designing your MinIO deployment to explore the effect of erasure code settings
 on your intended topology.
 
+Zero-Parity Deployments
+-----------------------
+
+Starting with :minio-release:`RELEASE.2022-06-02T02-11-04Z`, MinIO supports a Single-Node Single-Drive (SNSD) topology with a zero-parity erasure coding backend. 
+This provides access to erasure coding features while deferring any and all drive-level resiliency to the underlying storage volume.
```

Comment:
Strike `any and all`. It's unnecessary.

Reply:
Took your suggestion in the next comment, which removes the need to fix this one.